src/index.ts

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("dotenv-safe").config();
+
+import express from "express";
+
+const PORT = process.env.PORT || 5000;
+
+const app = express();
+
+app.get("/", (req, res) => {
```

Comment:
eslint should show a warning saying the parameter is not used. Can you prefix the parameter name with underscore to suppress it @CowboyBebop ?

Reply:
No problemo 😉